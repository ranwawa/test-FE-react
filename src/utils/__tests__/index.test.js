import { isMobile, REG_MOBILE } from '..';

describe('验证手机号码函数相关测试', () => {
  test('输入正确的手机号码:13333333333,应该返回true', () => {
    const res = isMobile('13333333333');
    expect(res).toBe(true);
  });

  test('输入错误的手机号码:1333333,应该返回false', () => {
    const res = isMobile('1333333');
    expect(res).toBe(false);
  });

  test('手机号码的正则表达式应该是11位数字', () => {
    expect(REG_MOBILE).toMatchSnapshot();
  });
});
