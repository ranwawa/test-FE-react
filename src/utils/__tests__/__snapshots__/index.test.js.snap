// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`验证手机号码函数相关测试 手机号码的正则表达式应该是11位数字 1`] = `/1\\[3-8\\]\\\\d\\{9,9\\}/`;
